src/content-service/services/api-provider.ts

```diff
@@ -57,6 +57,18 @@ export const getContentServiceApi = (
     return data;
   };
 
+  const getProductFromV3 = async (id: string): Promise<ApiProductDataV3> => {
+    const requestUrl = `${productsPath}/${id}?getContentFromV3=true`;
```

Comment:
this path need to be adjusted to hit the right version of the service:
`https://content-service.todaytix.com/api/v3/products/34579?includeAllFields=true'

Also we can remove `getContentFromV3` param and add `includeAllFields=true` instead.

Reply:
pushed new code, pls have a look if I am in the right direction